content/blog/2022/2022-04-20-KaplanMeier.md

```diff
@@ -0,0 +1,73 @@
+
+What would happend if we would ignore censored data? We could just exclude them from the analysis but we don't want to do that, because these data points are still informative when it comes to survival. And if we would treat them as surviving we would overestimate the survival probability. 
+
+If we plot the survival probability for each day we get the Kaplan-Meier Plot. 
+
+
+**Orange usecase**
+
+Now that we calculated the Kaplan-Meirer estimator on a small dataset manually and got an understanding what's going on, let's perform this on a larger dataset in Orange. We will use the data from the German Breast Cancer Study Group 2, a survival dataset that is made readily available in Orange. First we load the data using the **Datasets** widget and inspect the data in tabular format with the use of **Data Table**. We see that our data consists of 686 data instances that stand for patients. Each patient is characterized by 8 clinical features and the time at which reacurrence of the disease happened or when the patient stopped being part of the experiment due to unknown reasons (censored). We then connect our data with the **Kaplan-Meier widget** that calculates the survival probability at different time points and plots the survival curve. The widget also allows us to choose one of the three categorical features and compare the survival curves of the different categories.
+
+![](/orange-hugo/static/blog_img/2022/2022-04-20-kmworkflow.png)
+
+Comparing two or more survival curves can be done by visual inspection or by comparing different statistics. For instance we can compare the survival curves of patients according to their tumor grade which is a quantative measure of how abnormal the tumor cells and tissue look under a microscope. For breast cancer doctors most often use the Nottingham grading system which divides the tumor into three grades, where 1 coresponds to low grade, 2 to intermediate grade and 3 to high grade. The high grade means that the cells and tissue look most abnormal compared to healthy tissue. 
+
+We can see this reflected in the survival curves of the three grades: patients with grade 1 tumors have the highest survival probability throughout the duration of the study and patients with grad 3 the lowest. We can also plot the confidence intervals and check if they overlap. The confidence intervals of all three survival curves are seperated for the first two years (700 days), meaning that there is probably a significant difference in reacurrence of breast cancer between different tumor grades in the first two years. 
```

Comment:
with grad 3 --> with grade 3
curves are seperated --> curves are separated
reacurrence of breast --> re-occurrence of breast

Reply:
Or recurrence. Whatever you prefer or is more exact.